deltacat/compute/compactor/model/repartition_result.py

```diff
@@ -0,0 +1,6 @@
+from typing import NamedTuple, List
```

Comment:
If you agree this is a separate compute job in DeltaCAT, should we create a separate module e.g., `compute/repartition`?

Reply:
good idea. let's stay with the current structure as it's still high customized to the compaction use case. In future, especially on glue, we may expand to more generic API.

Reply:
added a TODO